MediaBrowser.Providers/Plugins/Tmdb/TmdbClientManager.cs

```diff
@@ -148,8 +148,10 @@ public async Task<TvShow> GetSeriesAsync(int tmdbId, string language, string ima
         private async Task<TvGroupCollection> GetSeriesGroupAsync(int tvShowId, string displayOrder, string language, string imageLanguages, CancellationToken cancellationToken)
         {
             TvGroupType? groupType =
+                string.Equals(displayOrder, "production", StringComparison.Ordinal) ? TvGroupType.Production :
                 string.Equals(displayOrder, "absolute", StringComparison.Ordinal) ? TvGroupType.Absolute :
                 string.Equals(displayOrder, "dvd", StringComparison.Ordinal) ? TvGroupType.DVD :
+                string.Equals(displayOrder, "tv", StringComparison.Ordinal) ? TvGroupType.TV :
```

Comment:
What is TV order?

Reply:
As far as I know it's supposed to be used for shows with episodes that were grouped differently in specific regions, yet unfortunately it's not a very consistent option:

- [Naruto](https://www.themoviedb.org/tv/46260) shows a five-season grouping as opposed to other options that show either a four-season grouping or a single season with all episodes lumped in.
- [One Piece](https://www.themoviedb.org/tv/37854) has all its episodes grouped into a single season in TV order, while other options are far better organized.

Reply:
In that case I think "tv" might be a bad name. Not sure what it should be called though.

Reply:
Neither do I in that case, the names used by the website are as descriptive as they get.

Reply:
Can we call it "regional"?

Reply:
I think it does, but it would be inconsistent and confusing compared to the TMDB name

Reply:
Is not "TV" = "Syndication"? See https://en.wikipedia.org/wiki/Broadcast_syndication
Why not add other types? For example "La casa de papel" (Money Heist) had Season 1 Spain version (aired, 15 episodes) and Netflix version (digital, 22 episodes) with different durations and number.

Reply:
The other types I hadn't added in this PR have obvious names (OriginalAirDate, StoryArc, Digital) so maybe just calling TV "Syndication" would be enough to have all of them?

Reply:
Why not list all types and in the same order [as here](https://github.com/LordMike/TMDbLib/blob/ef5d852982422f95fb646034c56d9dc23ce40f27/TMDbLib/Objects/TvShows/TvGroupType.cs)?

Reply:
I have pushed a commit listing all the types found in the client API.

Reply:
I think using the values provided by the API are sensible, if we rename them then it's on us to come up with better mappings.